packages/core/types/common.d.ts

```diff
@@ -94,16 +94,6 @@ export interface Session {
   user?: object;
 }
 
-export interface NotifyOpts {
-  context?: string;
-  device?: object;
-  request?: object;
-  user?: object;
-  metaData?: object;
-  severity?: "info" | "warning" | "error";
-  onError?: OnError;
-}
```

Comment:
hmm, it should have happened in #655 – I must have messed up somehow. Since #655 is already merged `NotifyOpts` is no longer used but I can split it out into a separate PR if you prefer?

Reply:
Can leave it in this PR for me, just wanted to be sure it was right.